include/git2/sys/transport.h

```diff
@@ -25,6 +25,24 @@
 
 GIT_BEGIN_DECL
 
+/**
+ * Flags to pass to transport
+ *
+ * Currently unused.
+ */
+typedef enum {
+	GIT_TRANSPORTFLAGS_NONE = 0,
+} git_transport_flags_t;
```

Comment:
Dead code? This will emit a warning on strict compilers.

Reply:
Thanks, removed.